### PR TITLE
fix autocorrect range

### PR DESCRIPTION
**Pull request by josoriom**

### CODE REVIEW

File: src/parse/swissDrivingLicenseFields.ts [hunk 1]
```diff
@@ -17,72 +17,83 @@ import {
...
-    line: 1,
-    start: 25,
-    end: 30,
-    parser: checkSeparator,
-  },
-  { ...lastNameTemplate, line: 2, start: 0, end: 30 },
-  { ...firstNameTemplate, line: 2, start: 0, end: 30 },
-].map(createFieldParser);
+export default function swissDrivingLicenseFields(lines: string[]) {
+  const names = lines[2].replace(/^<+|<+$/g, '').split('<<');
```
Comment (targos) [inline src/parse/swissDrivingLicenseFields.ts:21]: This is very risky as it only works if the line is formatted correctly

Comment (josoriom) [reply src/parse/swissDrivingLicenseFields.ts:21]: @targos It is acceptable to use "<<" as the delimiter between the first and last name?

Comment (targos) [reply src/parse/swissDrivingLicenseFields.ts:21]: Yes, but the parsing should be resilient to lines that don't follow the standard.
- What about lines without any `<<`?
- What about lines with multiple names separated by two `<<`?

I would prefer not to fundamentally change the parser to accomodate an edge case related to autocorrect only.

Also, this pattern isn't specific to the Swiss driving license. Don't we need similar code changes for other MRZ formats?

**Comment by targos** [comment]
Here I propose a fix that doesn't change how the parsing works: https://github.com/cheminfo/mrz/pull/45

**Comment by josoriom** [comment]
This solution addresses the problem and enhances error handling flexibility. Thanks!

**Pull request closed without merging**